attestation/product/product.go

```diff
@@ -103,6 +103,16 @@ type Attestor struct {
 	compiledExcludeGlob glob.Glob
 }
 
+type attestorJson struct {
+	Products      map[string]attestation.Product `json:"products"`
+	Configuration attestorConfiguration          `json:"configuration"`
```

Comment:
```suggestion
	Configuration *attestorConfiguration          `json:"configuration,omitempty"`
```

I think it would be nice to exclude the config if no globs are set.